cachito/web/static/api_v1.yaml

```diff
@@ -558,6 +568,7 @@ paths:
                     example: The logs for the Cachito request 1 no longer exist
   "/requests/{id}/packages":
     get:
+      summary: List request packages and dependencies
```

Comment:
Maybe it would be less ambiguous if we kept it as `List packages and dependencies for a request`?

Reply:
fixed